app.js

```diff
@@ -0,0 +1,22 @@
+const schedule = require('node-schedule');
+const requireEnv = require('require-environment-variables');
+
+const updateElasticSearch = require('./lib/updateElasticsearch');
+const log = require('./lib/utils/logger');
+const config = require('./config/config');
+
+requireEnv(['JSON_FILE_URL']);
+requireEnv(['ES_HOST']);
+
+async function runUpdater() {
+  // run on initial start, then on the schedule
+  await updateElasticSearch();
+
+  log.info(`Scheduling Elasticsearch update with rule '${config.updateSchedule}'`);
+  schedule.scheduleJob(config.updateSchedule, () => {
+    // this is an async function, but await is not allowed within a lambda expression
```

Comment:
Lambda's are good for terseness but it doesn't preclude using a function if there are other factors such as conveying intent without comments.